dev/conductor/core/lib/src/next.dart

```diff
@@ -250,83 +247,18 @@ class NextContext extends Context {
-        final Remote engineUpstream = Remote(
-            name: RemoteName.upstream,
-            url: state.engine.upstream.url,
-        );
-        final EngineRepository engine = EngineRepository(
-            checkouts,
-            // We explicitly want to check out the merged version from upstream
-            initialRef: '${engineUpstream.name}/${state.engine.candidateBranch}',
-            upstreamRemote: engineUpstream,
-            previousCheckoutLocation: state.engine.checkoutPath,
-        );
-        final String engineHead = await engine.reverseParse('HEAD');
-        if (!autoAccept) {
-          final bool response = await prompt(
-            'Are you ready to tag commit $frameworkHead as ${state.releaseVersion}\n'
-            'and push to remote ${state.framework.upstream.url}?',
-          );
-          if (!response) {
-            stdio.printError('Aborting command.');
-            updateState(state, stdio.logs);
-            return;
-          }
-        }
-        await framework.tag(frameworkHead, state.releaseVersion, frameworkUpstream.name);
-        await engine.tag(engineHead, state.releaseVersion, engineUpstream.name);
-      case pb.ReleasePhase.PUBLISH_CHANNEL:
-        final Remote upstream = Remote(
-            name: RemoteName.upstream,
-            url: state.framework.upstream.url,
-        );
-        final FrameworkRepository framework = FrameworkRepository(
-            checkouts,
-            // We explicitly want to check out the merged version from upstream
-            initialRef: '${upstream.name}/${state.framework.candidateBranch}',
-            upstreamRemote: upstream,
-            previousCheckoutLocation: state.framework.checkoutPath,
-        );
-        final String headRevision = await framework.reverseParse('HEAD');
-        if (!autoAccept) {
-          // dryRun: true means print out git command
-          await framework.pushRef(
-              fromRef: headRevision,
-              toRef: state.releaseChannel,
-              remote: state.framework.upstream.url,
-              force: force,
-              dryRun: true,
-          );
-
-          final bool response = await prompt(
-            'Are you ready to publish version ${state.releaseVersion} to ${state.releaseChannel}?',
-          );
-          if (!response) {
-            stdio.printError('Aborting command.');
-            updateState(state, stdio.logs);
-            return;
-          }
-        }
-        await framework.pushRef(
-            fromRef: headRevision,
-            toRef: state.releaseChannel,
-            remote: state.framework.upstream.url,
-            force: force,
-        );
+        final String command = '''
+          tool-proxy-cli --tool_proxy=/abns/dart-eng-tool-proxy/prod-dart-eng-tool-proxy-tool-proxy.annealed-tool-proxy \\
+          --block_on_mpa -I flutter_release \\
+          :git_branch ${state.framework.candidateBranch} \\
```

Comment:
this tool uses colons in front of CLI options?

Reply:
Yep! 😢